firmware-nucleo/smart-air-hockey-table/App/driver_display.c

```diff
@@ -115,7 +114,7 @@ void Driver_Display_Print(Display display, uint8_t line, uint8_t offset, const c
 
 	// Send string to the display
 	bool inverted = 0;

```

Comment:
please make this `false` lol. If we are using bool types we should use them well haha

Reply:
my todo later

Reply:
That must've been missed in the original change, I'll get it this time.